src/config/HttpCode.ts

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable no-unused-vars */
+export enum HttpStatusCode {
```

Comment:
Why did you list everything, list only the thing you need

More over, Http status codes are something that every web developer should know, therefor no need for an enum :)